examples/scene_side_by_side.py

```diff
@@ -49,11 +49,9 @@ def animate():
     rot = gfx.linalg.Quaternion().set_from_euler(gfx.linalg.Euler(0.005, 0.01))
     cube1.rotation.multiply(rot)
 
-    size = canvas.get_logical_size()
-    renderer.render(scene1, camera1, flush=False, region=(0, 0, size[0] / 2, size[1]))
-    renderer.render(
-        scene2, camera2, flush=False, region=(size[0] / 2, 0, size[0] / 2, size[1])
-    )
+    w, h = canvas.get_logical_size()
+    renderer.render(scene1, camera1, flush=False, region=(0, 0, w / 2, h))
+    renderer.render(scene2, camera2, flush=False, region=(w / 2, 0, w / 2, h))
     renderer.flush()
```

Comment:
why are you calling flush manually here? :)

Reply:
Then it's more of a pattern of rendering N subplots and flush at the end. Lemme make it 4 subplots so that its a bit more obvious.

Reply:
Right, so it's just one way to do it that makes sense stylistically for this example, and it works. :)

Reply:
Exactly. I thought about making flush private, but I think I might like this style too.